### PR TITLE
Deprecated args in examples

### PR DESCRIPTION
**Pull request by wenhuach21**
three args has been set to True in examples to align the api
amp, low_gpu_mem_usage, eanble_minmax_tuning

### CODE REVIEW

File: examples/language-modeling/main.py [hunk 1]
```diff
@@ -100,25 +103,53 @@
...
                         help="Whether to do lmeval evaluation.")
 
+    parser.add_argument("--disable_amp", action='store_true',
+                        help="disable amp")
+
+    parser.add_argument("--disable_low_gpu_mem_usage", action='store_true',
+                        help="disable low_gpu_mem_usage")
+
+    parser.add_argument("--disable_minmax_tuning", action='store_true',
+                        help="disable whether enable weight minmax tuning")
```
Comment (WeiweiZhang1) [inline examples/language-modeling/main.py]: should be 'whether disable weight minmax tuning'

File: examples/code-generation/main.py [hunk 2]
```diff
@@ -96,32 +95,53 @@
...
                         help="Where to store the final model.")
 
+    parser.add_argument("--disable_amp", action='store_true',
+                        help="disable amp")
+
+    parser.add_argument("--disable_low_gpu_mem_usage", action='store_true',
+                        help="disable low_gpu_mem_usage")
+
+    parser.add_argument("--disable_minmax_tuning", action='store_true',
+                        help="disable whether enable weight minmax tuning")  # #not usage
```
Comment (WeiweiZhang1) [inline examples/code-generation/main.py]: 'whether disable weight minmax tuning'

File: examples/language-modeling/README.md [hunk 3]
```diff
@@ -47,25 +47,27 @@ pip install -r requirements.txt
 
 - **Default Settings:**
 ```bash
-CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1 --enable_minmax_tuning --use_quant_input
+CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1  --use_quant_input
 ```
 - **Reduced GPU Memory Usage and Adjusted Training Batch Size:**
 ```bash
-CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1 --low_gpu_mem_usage --train_bs 1 --gradient_accumulate_steps 8
+CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1  --train_bs 1 --gradient_accumulate_steps 8
```
Comment (WeiweiZhang1) [inline examples/language-modeling/README.md]: remove '--amp'

File: examples/language-modeling/README.md [hunk 4]
```diff
@@ -47,25 +47,27 @@ pip install -r requirements.txt
 
 - **Default Settings:**
 ```bash
-CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1 --enable_minmax_tuning --use_quant_input
+CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1  --use_quant_input
```
Comment (WeiweiZhang1) [inline examples/language-modeling/README.md]: remove '--amp'

File: examples/language-modeling/scripts/Mixtral-8x7B-v0.1.sh [hunk 5]
```diff
@@ -4,10 +4,6 @@ python3 main.py \
 --group_size 128 \
 --bits 4 \
 --iters 1000 \
---enable_minmax_tuning \
---low_gpu_mem_usage \
 --deployment_device 'gpu' \
 --scale_dtype 'fp32' \
```
Comment (WeiweiZhang1) [inline examples/language-modeling/scripts/Mixtral-8x7B-v0.1.sh]: scale_dtype is removable

File: examples/language-modeling/README.md [hunk 6]
```diff
@@ -47,25 +47,27 @@ pip install -r requirements.txt
...
 ```
 - **Reduced GPU Memory Usage and Adjusted Training Batch Size:**
 ```bash
-CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1 --low_gpu_mem_usage --train_bs 1 --gradient_accumulate_steps 8
+CUDA_VISIBLE_DEVICES=0 python3 main.py --model_name facebook/opt-125m --amp --bits 4 --group_size -1  --train_bs 1 --gradient_accumulate_steps 8
 ```
 - **Utilizing the AdamW Optimizer:**
+
 Include the flag `--adam`. Note that AdamW is less effective than Sign gradient descent in many scenarios we tested.
```
Comment (WeiweiZhang1) [inline examples/language-modeling/README.md]: should be 'sign'

**Review by WeiweiZhang1 — APPROVED** [review]
